lib/json_serializable/serializable_complex_object.dart

```diff
@@ -0,0 +1,35 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:json_annotation/json_annotation.dart';
+import 'package:jsonexample/json_serializable/serializable_simple_object.dart';
```

Comment:
Flutter is all about full package URIs, right? *sigh* Unfortunate...

Reply:
This is more of a personal choice. If I always use package-style imports, I can't screw up and double-namespace something accidentally. :)